src/components/home/hero/hero.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import Background from '../../../assets/background';
```

Comment:
There's a fun thing you can do in your tsconfig to make these routes much easier to manage.

```
        "baseUrl": "./src",
        /* Base directory to resolve non-absolute module names. */
        "paths": {
            "~/*": ["./*"]
        }
```

This will let you do...

```
import Background from '~/assets/background';
```

Reply:
Thank fucking god